### PR TITLE
Layout and update category

### PR DESCRIPTION
**Pull request by mvn-haphan-dn**
## Description
Layout and update category
## Reference Requirements
- Task ID: TI-78

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)
```
explain select count(*) as aggregate from `categories` where `name` = 'Dr. Rickey Greenholt IV'\G
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: categories
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 20
     filtered: 10.00
        Extra: Using where
```
```
explain select * from `categories` where `categories`.`id` = '1' limit 1\G
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: categories
   partitions: NULL
         type: const
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: NULL
```
```
explain update `categories` set `name` = 'Dr. Rickey Greenholt IV', `updated_at` = '2018-06-01 09:34:26' where `id` = '1'\G
*************************** 1. row ***************************
           id: 1
  select_type: UPDATE
        table: categories
   partitions: NULL
         type: range
possible_keys: PRIMARY
          key: PRIMARY
      key_len: 4
          ref: const
         rows: 1
     filtered: 100.00
        Extra: Using where
```
- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Admin/CategoryController.php [hunk 1]
```diff
@@ -30,8 +30,25 @@ public function index()
...
+     * @param App\Models\Category           $id      id of Category
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function update(CategoryRequest $request, $id)
+    {
+        $category = Category::findOrFail($id);
+        $category->name = $request->name;
+        $category->save();
+        return response()->json(['category' => $category,'msg' => trans('category.messages.update_success')]);
```
Comment (at-haytran) [inline app/Http/Controllers/Admin/CategoryController.php]: cách sau dấu ,

File: resources/views/admin/categories/index.blade.php [hunk 2]
```diff
@@ -43,19 +44,41 @@
...
+                    <div class="box-body">
+                        <div class="form-group">
+                            <label>{{trans('category.form.title_inputs.name')}}</label>
+                            <input type="text" class="form-control" id="category-update" placeholder=" {{trans('category.placeholders.name')}}" value="" name="categoryName">
+                        </div>
+                    </div>
+                    <!-- /.box-body -->
+                    <div class="box-footer">
+                        <button type="submit" name="update-btn" class="btn btn-primary">{{trans('category.buttons.update')}}</button>
+                        <button type="reset" class="btn">{{trans('category.buttons.reset')}}</button>
```
Comment (at-haytran) [inline resources/views/admin/categories/index.blade.php]: space

File: resources/views/admin/categories/index.blade.php [hunk 3]
```diff
@@ -43,19 +44,41 @@
...
+                    {{ csrf_field() }}
+                    <div class="box-body">
+                        <div class="form-group">
+                            <label>{{trans('category.form.title_inputs.name')}}</label>
+                            <input type="text" class="form-control" id="category-update" placeholder=" {{trans('category.placeholders.name')}}" value="" name="categoryName">
+                        </div>
+                    </div>
+                    <!-- /.box-body -->
+                    <div class="box-footer">
+                        <button type="submit" name="update-btn" class="btn btn-primary">{{trans('category.buttons.update')}}</button>
```
Comment (at-haytran) [inline resources/views/admin/categories/index.blade.php]: space

**Comment by at-haytran** [comment]
LGTM

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**